pimd/pim6_cmd.c

```diff
@@ -451,6 +451,34 @@ DEFPY (no_ipv6_pim_rp_prefix_list,
 	return pim_process_no_rp_plist_cmd(vty, rp_str, plist);
 }
 
+
+DEFPY (ipv6_ssmpingd,
+      ipv6_ssmpingd_cmd,
+      "ipv6 ssmpingd [X:X::X:X]$source",
+      IPV6_STR
+      "Enable ssmpingd operation\n"
+      "Source address\n")
+{
+
+	const char *src_str = (source_str) ? source_str : "::";
+
+	return pim_process_ssmpingd_cmd(vty, NB_OP_CREATE, src_str);
+}
+
+
+DEFPY (no_ipv6_ssmpingd,
+      no_ipv6_ssmpingd_cmd,
+      "no ipv6 ssmpingd [X:X::X:X]$source",
+      NO_STR
+      IPV6_STR
+      "Enable ssmpingd operation\n"
```

Comment:
Please use CONF_SSMPINGD_STR instead of "Enable ssmpingd operation\n"

Reply:
@g-balaji1 Please address the above comments. Other things look fine.

Reply:
Ack